cli/flags.rs

```diff
@@ -3474,6 +3474,29 @@ mod tests {
     );
   }
 
```

Comment:
I think to have more check , we should add check for single test watch 

```suggestion
  #[test]
  fn test_single_file_watch() {
    let r = flags_from_vec(svec!["deno", "test", "--watch", "test/ignore.ts"]);
    assert_eq!(
      r.unwrap(),
      Flags {
        subcommand: DenoSubcommand::Test {
          no_run: false,
          doc: false,
          fail_fast: None,
          filter: None,
          allow_none: false,
          quiet: false,
          shuffle: None,
          include: Some(svec!["test/ignore.ts"]),
          concurrent_jobs: 1,
        },
        watch: true,
        ..Flags::default()
      }
    );
  }
```

Reply:
#11432 was caused by skipping the check of `watch` flag. The current change is enough for checking that in my view.